assets/src/components/shuttleMapPage.tsx

```diff
@@ -39,6 +49,14 @@ const ShuttleMapPage = ({}): ReactElement<HTMLDivElement> => {
   const { socket }: { socket: Socket | undefined } = useContext(SocketContext)
   const shuttles: Vehicle[] | null = useShuttleVehicles(socket)
   const shapes: Shape[] = useRouteShapes(selectedShuttleRouteIds)
+
+  const selectedSubwayRouteIds: RouteId[] = selectedShuttleRouteIds.filter(
+    isASubwayRoute
+  )
+  const trainVehicles: TrainVehicle[] = allTrainVehicles(
+    useTrainVehicles(socket, selectedSubwayRouteIds)
```

Comment:
This does meet the rules for calling hooks, since it's called in the same way every time, but it still surprised me to see a hook call inside a function argument instead of as a top level variable assignment.